modules/elasticsearch/templates/redirect.nomad

```diff
@@ -0,0 +1,81 @@
+job "${redirect_job_name}" {
+  datacenters = ${az}
+  region      = "${region}"
+  type        = "service"
+
+  update {
+    max_parallel     = 1
+    auto_revert      = true
+  }
+
+  group "${redirect_job_name}" {
+    count = 2
+
+    task "${redirect_job_name}" {
+      driver = "docker"
+
+      config {
+        image = "nginx:${redirect_nginx_version}"
+
+        port_map {
+          http = 80
+        }
+
+        volumes = [
+          "alloc/nginx.conf:/etc/nginx/conf.d/default.conf:ro",
+        ]
+
+        dns_servers = ["169.254.1.1"]
+
+        # Keep default logging -- nothing special to keep track of!
+      }
+
+      template {
+        data = <<EOH
+server {
+    listen       80;
+    server_name  localhost;
+
+    rewrite ^ https://{{ with service "${elasticsearch_service}" }}{{ (index . 0).Address }}{{ end }}/_plugin/kibana/;
+}
+EOH
+
+        destination = "alloc/nginx.conf"
+        change_mode = "restart"
+      }
+
+      service {
+        name = "$${JOB}"
+        port = "http"
+
+        check {
+          type     = "http"
+          path     = "/"
+          interval = "10s"
+          timeout  = "2s"
+        }
+
+        tags = [
```

Comment:
Can mention in the README that this job will be automatically reverse proxied by Traefik if they use the Traefik module.